src/liboslexec/llvm_util.cpp

```diff
@@ -449,6 +450,17 @@ LLVM_Util::make_jit_execengine (std::string *err)
     if (! m_llvm_exec)
         return NULL;
 
+    // These magic lines will make it so that enough symbol information
```

Comment:
Well that's easy :+1: 

Let me get a benchmark run setup with 5.0.0 vs this patch.

Reply:
Performance tests are looking good. No measurable change.

LGTM

Reply:
You mean no measurable change on 5.0.1, the one where we built LLVM with the support for this enabled, right?

Reply:
Correct.